tasks/export_tasks.py

```diff
@@ -0,0 +1,76 @@
+import json
+import sys
+
+import requests
+
+sys.path.insert(1, ".")
+from invoke import task  # noqa:E402
+
+from app.app import app  # noqa:E402
+from app.blueprints.self_serve.routes import human_to_kebab_case  # noqa:E402
+from app.config_generator.scripts.generate_fund_round_config import (  # noqa:E402
+    generate_config_for_round,
+)
+from app.config_generator.scripts.generate_fund_round_form_jsons import (  # noqa:E402
+    generate_form_jsons_for_round,
+)
+from app.config_generator.scripts.generate_fund_round_html import (  # noqa:E402
+    generate_all_round_html,
+)
+from config import Config  # noqa:E402
+
+
+@task
+def generate_fund_and_round_config(c, roundid):
+    if not roundid:
+        print("Round ID is required.")
+        return
+    print(f"Generating fund-round configuration for round ID: {roundid}.")
+    with app.app_context():
+        generate_config_for_round(roundid)
+
+
+@task
+def generate_round_form_jsons(c, roundid):
+    if not roundid:
+        print("Round ID is required.")
+        return
+    print(f"Generating form JSON configuration for round ID: {roundid}.")
+    with app.app_context():
+        generate_form_jsons_for_round(roundid)
+
+
+@task
+def generate_round_html(c, roundid):
+    if not roundid:
+        print("Round ID is required.")
+        return
+    print(f"Generating HTML for round ID: {roundid}.")
+    with app.app_context():
+        generate_all_round_html(roundid)
+
+
+@task
+def publish_form_json_to_runner(c, filename):
+
+    if not filename:
+        print("filename is required.")
+        return
+    print(f"Publishing: {filename} to form runner.")
+
+    # load json from filename
+    with open(filename, "r") as file:
+        form_json = file.read()
+        form_dict = json.loads(form_json)
+        form_dict["outputs"][0]["outputConfiguration"][
```

Comment:
For now we uses this but after the new adapter runner this part will not be using

Reply:
understood - there will be a ticket for E2E trial fund-round that will tackle this